app/controllers/application_controller.rb

```diff
@@ -132,6 +133,14 @@ def store_user_location!
     store_location_for(:user, request.fullpath)
   end
 
+  def updated_last_visited_on!
+    return unless user_signed_in?
+    return unless request.format == 'html'
```

Comment:
Not sure if this is the "proper" way to do it, but the aim was to not updated the visited on date when doing an API request

Reply:
I think request.format is a symbol, not a string (based on `config/initializers/devise.rb`)